### PR TITLE
Completed css-tailwind-intro exercise

### PR DESCRIPTION
**Pull request by Sam-Castaneda13**
Screenshot 2024-08-14 120908

### CODE REVIEW

**Review by brandon-moy — CHANGES_REQUESTED** [review]
It looks like you worked through the Tailwind example but you need to build out a page layout with a header, footer, and two content sections side by side (in the Exercise section)

**Comment by Sam-Castaneda13** [comment]
Oh my bad I will fix it now

**Comment by Sam-Castaneda13** [comment]
Screenshot 2024-08-14 123308

File: css-tailwind-intro/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      rel="stylesheet" />
+    <title>Tailwind CSS Website Layout</title>
+  </head>
+  <body class="bg-gray-100">
+    <div class="container mx-auto p-4">
+      <div class="header bg-blue-500 text-white p-4 rounded-lg shadow-lg">
+        <h1 class="text-3xl font-bold">My Website</h1>
+      </div>
+      <div class="flex mt4">
+        <div class="w-1/4 bg-gray-200 p-4 rounded-lg shadow-lg">
```
Comment (brandon-moy) [inline css-tailwind-intro/index.html:18]: Missed a `-` here for `mt-4`

**Review by brandon-moy — APPROVED** [review]
One minor typo but approved 👍

**Pull request merged**